demoapp/src/androidTest/java/me/uport/sdk/demoapp/ImportKeyTest.kt

```diff
@@ -0,0 +1,46 @@
+package me.uport.sdk.demoapp
+
+import android.support.test.espresso.Espresso.onView
+import android.support.test.espresso.action.ViewActions.click
+import android.support.test.espresso.assertion.ViewAssertions.matches
+import android.support.test.espresso.matcher.ViewMatchers.*
+import android.support.test.rule.ActivityTestRule
+import android.support.test.runner.AndroidJUnit4
+import org.hamcrest.CoreMatchers.not
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class ImportKeyTest {
+
+    @JvmField
+    @Rule
+    val activityRule = ActivityTestRule(ImportKeyActivity::class.java, true, false)
```

Comment:
if you use the simple form, without the boolean parameters `val activityRule = ActivityTestRule(ImportKeyActivity::class.java)`

you won't need to call `activityRule.launchActivity(null)` in `@Before`


It looks like there are no reasons to delay the activity launch so the simple form is preferred